packages/mdc-drawer/_variables.scss

```diff
@@ -41,5 +41,9 @@ $mdc-drawer-surface-padding: 16px;
 $mdc-drawer-animation-enter: 250ms;
 $mdc-drawer-animation-exit: 200ms;
 
-$mdc-drawer-z-index: 5;
+// Scrim
+$mdc-drawer-modal-scrim-color: #000 !default;
```

Comment:
I think we'd want to encourage use of the mixin rather than make these overridable. We're pretty selective about adding `!default` and it tends to more often apply to systems.